src/screens/ImportMethod/ImportMethod.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import React, { PureComponent } from 'react'
+import {
+  Image,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native'
+import textConstants from '../../locales/en'
+import PropTypes from 'prop-types'
+import TextButton from '../../components/TextButton'
+import styles from './ImportMethodStyles'
+
+
+
+const Method = ({image, label, onPress}) => {
+    return (
+      <TouchableOpacity
+        onPress={onPress}
+        style={styles.item}
+      >
+        <Image
+          source={image}
+          style={styles.itemImage}
+        />
+        <Text style={styles.itemLabel}>
+          {
+            label
+          }
+        </Text>
+      </TouchableOpacity>
+    )
+}
+
+Method.propTypes = {
+  label: PropTypes.string,
+  onPress: PropTypes.func,
+}
+
+export default class ImportMethod extends PureComponent {
+  renderMethod = (ImportMethod) => (
+    <Method
+      key={ImportMethod.id}
+      {...ImportMethod}
+      onPress={this.props.onSelectImportMethod(ImportMethod)}
+    />
+  )
+
+  render() {
+    const {
+      ImportMethods,
```

Comment:
Code style:
all variables must be named in camel-case and starts with small character. In this case it must be importMethods.
But even better to call it `importMethodList` (with no 's' in the middle)